src/components/FilterGroup.tsx

```diff
@@ -75,11 +79,17 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const collapsibleLabelCssClasses: CollapsibleLabelCssClasses = useMemo(() => {
+    return {
+      label: cssClasses.titleLabel
+    };
+  }, [cssClasses]);
+
   function renderTitle() {
     return collapsible
-      ? <CollapsibleLabel label={title} />
+      ? <CollapsibleLabel label={title} customCssClasses={collapsibleLabelCssClasses} />
       : (title &&
-        <div className='text-neutral-dark text-sm font-medium text-left mb-4'>
+        <div className={twMerge('mb-4', builtInCollapsibleLabelCssClasses.label, cssClasses.titleLabel)}>
```

Comment:
you may want this `twMerge` in a useMemo as well then pass that memoized value to className

Reply:
could use `useComposedCssClasses` instead of directly calling twMerge